Initial checkin of telemetry FAQ.

geeknoid opened the pull request:
This is a markdown version of the telemetry FAQ.

istio-testing commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio.github.io/pull/2024#" title="Author self-approved">geeknoid</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[content/OWNERS](https://github.com/istio/istio.github.io/blob/master/content/OWNERS)~~ [geeknoid]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

geeknoid commented:
@sdake Thanks for the review, I addressed your feedback, except for the I->you thing. For the FAQ, these are supposed to be questions the user is asking him/herself.

sdake commented:
/lgtm
/approve

istio-testing commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/istio/istio.github.io/pull/2024#" title="Author self-approved">geeknoid</a>*, *<a href="https://github.com/istio/istio.github.io/pull/2024#issuecomment-408853482" title="Approved">sdake</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[content/OWNERS](https://github.com/istio/istio.github.io/blob/master/content/OWNERS)~~ [geeknoid,sdake]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->
